### PR TITLE
Quiz moodle joneice clean

### PR DESCRIPTION
**Pull request by JoneiceB**
This is my course list UI

### CODE REVIEW

File: teamA/lib/Views/course_list.dart [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+                          fontSize: 20,
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              )),
+            ),
+            body: Padding(
```
Comment (DavidWorthy) [inline teamA/lib/Views/course_list.dart]: Add a SingleChildScrollView here. While it's unlikely that anyone will have enough courses for vertical space to be an issue, better safe than sorry.

**Review by DavidWorthy — CHANGES_REQUESTED** [review]
Sorry for taking so long, but here is my (one) suggestion on improvement. Otherwise, this looks good.

**Comment by JoneiceB** [comment]
I have updated the UI to reflect the suggestion.

**Comment by DavidWorthy** [comment]
Closed due to contributing element of this branch (the course list UI) having been added in another pull request.

**Pull request closed without merging**